packages/cli/src/cmds/permission.ts

```diff
@@ -17,13 +29,40 @@ import activate from "../activate";
 import { argsToInputs, getSystemInputs } from "../utils";
 import HelpParamGenerator from "../helpParamGenerator";
 import CLILogProvider from "../commonlib/log";
+import { FxCore } from "@microsoft/teamsfx-core";
 
 const azureMessage =
   "Notice: Azure resources permission needs to be handled by subscription owner since privileged account is " +
   "required to grant permission to Azure resources.\n" +
   "Assign Azure roles using the Azure portal: " +
   "https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=current";
 
+const spfxMessage =
+  "Notice: SPFX deployment permission needs to be handled manually by SharePoint site administrator.\n" +
+  "Manage site admins using SharePoint admin center: " +
+  "https://docs.microsoft.com/en-us/sharepoint/manage-site-collection-administrators";
+
+async function isSpfxProject(
+  rootFolder: string,
+  core: FxCore
+): Promise<Result<boolean | undefined, FxError>> {
```

Comment:
Is this better to add into core/solution? I think CLI/VSCode should not manage this hardcode logic.

Reply:
Thanks, updated and use an existing core function for this purpose

Reply:
Add two tasks in our board to optimize this in the future
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12763877
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12763884